src/lizaalert/homeworks/urls.py

```diff
@@ -0,0 +1,12 @@
+from django.urls import include, path
+from rest_framework import routers
+
+from lizaalert.homeworks.views import HomeworkViewSet
+
+router = routers.SimpleRouter()
+router.register(r"lessons/(?P<lesson_id>\d+)/homeworks", HomeworkViewSet, basename="homework")
+
+
+urlpatterns = [
+    path("", include(router.urls)),
+]

```

Comment:
```suggestion
from lizaalert.homeworks.views import HomeworkViewSet

urlpatterns = [
    path('lessons/<int:lesson_id>/homework/', HomeworkViewSet.as_view({
        'get': 'retrieve', 'post': 'create'}), name='lesson-homework-detail'),
]

```
К сожалению, из-за того что у нас нестандартные ручки, здесь не получится ограничится роутером, придется сделать немного танцев с бубнами, чтобы наши эндпоинты имели вид:
lesson/id/homework - retrieve
lesson/id/homework - post